### ISSUE TITLE
[Content Author Module 6: Multimedia] Typo found in Success Critieria

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-curricula#548]
WCAG Success Criterion 1.2.6 Sign Language has a typo with "prerecorded" -- the P needs to be capitalized and it's missing the closing parenthesis. 

> WCAG Success Criterion 1.2.6 Sign Language (prerecorded
